apps/io-services-cms-webapp/db/schema/migrations/reviewer/V6__create_service_topic_table.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE IF NOT EXISTS taxonomy.topic (
+	"id" smallint NOT NULL GENERATED BY DEFAULT AS IDENTITY (sequence name topic_id_seq minvalue 0), -- 'the unique identifier for the Topic'
+	"name" varchar NOT NULL, -- 'the name of the Topic'
+	"disabled" boolean NOT NULL, -- 'the name of the Topic'
```

Comment:
it's definitely a typo, should we change it to "topic enablement or visibility"?